src/components/Fund/FundForm/FundForm.js

```diff
@@ -354,13 +355,15 @@ FundForm.propTypes = {
   pristine: PropTypes.bool.isRequired,
   submitting: PropTypes.bool.isRequired,
   values: PropTypes.object.isRequired,  // current form values
+  systemCurrency: PropTypes.string,
 };
 
 FundForm.defaultProps = {
   initialValues: {
     fund: {},
     groupIds: [],
   },
+  systemCurrency: '',
```

Comment:
currency as empty string makes no sense, IMHO

Reply:
Removed it